sdk-lib/src/main/rust/lib.rs

```diff
@@ -733,17 +751,21 @@ pub unsafe extern "C" fn Java_cash_z_ecc_android_sdk_jni_RustBackend_getTotalTra
         let addr = utils::java_string_to_rust(&env, address);
         let taddr = TransparentAddress::decode(&network, &addr).unwrap();
 
+        // We select all transparent funds including unmined coins, as that is the same
+        // set of UTXOs we shield from.
+        let min_confirmations = 0;
```

Comment:
What kind of implications does this have for clients of the SDK?

Reply:
None. This change is a no-op; it just uses the `WalletRead` API correctly.